EPFParser.py

```diff
@@ -135,7 +126,19 @@ def __init__(self, filePath, typeMap={"CLOB":"LONGTEXT"}, recordDelim='\x02\n',
                 self.dataTypes = ['DECIMAL(11,3)' if dt == 'DECIMAL(9,3)' else dt for dt in dts]
             elif aRow.startswith(exStart):
                 self.exportMode = self.splitRow(aRow, requiredPrefix=exStart)[0]
-        self.eFile.seek(0, os.SEEK_SET) #seek back to the beginning
+
+        # Close and reopen as we don't have seek for a streams - and don't need it.
+        # self.eFile.seek(0, os.SEEK_SET) #seek back to the beginning
+        self.eFile.close()
+        self.rawFile.close()
+        self.eFile = None
```

Comment:
I don't think we need to set these to None, since they're replaced immediately.
Closing is necessary of course.
Are we missing a `self.bzFile.close()`, or does the BufferedReader chain through?

Reply:
BufferedReader seems to chain through, but this was to make sure the objects are destructed just in case. It's probably not needed, you're right.